tycho-compiler-plugin/src/main/java/org/eclipse/tycho/compiler/AbstractOsgiCompilerMojo.java

```diff
@@ -851,14 +855,22 @@ private void configureBootClassPath(CompilerConfiguration compilerConfiguration,
                 if (includeParent != null) {
                     Xpp3Dom[] includes = includeParent.getChildren("include");
                     if (includes.length > 0) {
-                        compilerConfiguration.addCompilerCustomArgument("-bootclasspath", scanBootclasspath(
+                        addCompilerCustomArgument(compilerConfiguration, "-bootclasspath", scanBootclasspath(
```

Comment:
Bootclasspath is a standard javac option. Skipping this might cause surprises.

Reply:
It seems not to be a standard plexus compiler option so I'm not sure how to handle this gracefully...